docs/index.md

```diff
@@ -9,9 +9,20 @@ title: Zaak- en Documentservices
 **Beheerder:**  VNG Realisatie<br/>
 **Afkorting:**  StUF-ZDS
 
-***Gemeenten, leveranciers en VNG Realisatie werken aan een nieuwe
+## Status en doorontwikkeling
+Verschillende ontwikkelingen maken het vernieuwen van standaarden noodzakelijk 
+om invulling te (blijven) geven aan de behoefte en wensen van gemeenten. 
+Gemeenten, leveranciers en VNG Realisatie werken daarom aan een nieuwe
 generatie standaarden voor zaakgericht werken. Zie voor de voortgang van
-dit project [hier](https://github.com/vng-realisatie/gemma-zaken).***
+dit project [hier](https://vng-realisatie.github.io/gemma-zaken/). 
```

Comment:
Volgens de digitaal toegankelijksrichtlijnen mogen verwijzigingen met het woord "hier" niet gebruikt worden.  Graag de tekst aanpassen naar "
Op de [GitHub-pagina van GEMMA-Zaken](https://vng-realisatie.github.io/gemma-zaken/) is de voortgang van het project te volgen.  
"